### ISSUE TITLE
brackets 1.10 not working on windows 7

### ISSUE THREAD

**Issue by Daryll0222** [adobe/brackets#13616]
### Prerequisites

* [ ] Can you reproduce the problem with `Debug -> Reload Without Extensions`?
* [ ] Did you perform a cursory search to see if your bug or enhancement is already reported?
* [ ] Did you read the [Troubleshooting guide](https://github.com/adobe/brackets/wiki/Troubleshooting)?

For more information on how to write a good bug report read [here](https://github.com/adobe/brackets/wiki/How-to-Report-an-Issue)
For more information on how to contribute read [here](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md)

### Description

brackets not working . the software is open but cant open file or close the app

### Steps to Reproduce

1. run brackets
2. open new file
3. cant open new file or open new folder or close brackets it is not working in windows 7

**Expected behavior:** opening file or folder

**Actual behavior:** nothing is happening except seeing a blank software

### Versions

brackets 1.10
windows 7

**Comment by petetnt** [comment]
Hi @Daryll0222,

can you see any errors in the console? `Debug -> Show Developer Tools -> Console`. Might be a permissions issue.

**Comment by teethnclaws** [comment]
I had to revert back to the 1.08 version of brackets in order to get my files to open and be recognized by the text editor again. I'm using windows 10; recently updated to brackets version 1.10 from 1.08 and encountered a similar problem where while my files appeared in the tree alongside the working area, however, they would not open or run. I had to revert back to 1.08 in order to get my brackets to work.

**Comment by Daryll0222** [comment]
@teethnclaws yeah thats what I did. I download brackets 1.09 to get what I want in 1.10 similar.. so there is no possible fix for us in 1.10?
@petetnt sorry dude i just got my brackets 1.09 before you comment. but thanks anyways .

**Comment by petetnt** [comment]
You can also try deleting Brackets and reinstalling Brackets via the latest installer. There were some backwards incompatible changes to file handling which require the new version of Brackets Shell. Sorry for you troubles!

**Comment by Daryll0222** [comment]
@petetnt yeah I already tried that before I go asking here .. no problems mates!
